src/Apps/Artist/Routes/Consign/Components/ArtistConsignHeader.tsx

```diff
@@ -1,19 +1,41 @@
-import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { Box, Button, Flex, ResponsiveImage, Sans, Serif } from "@artsy/palette"
+import { sampleSize } from "lodash"
 import React from "react"
 
+import { Consign_artworksByInternalID } from "__generated__/Consign_artworksByInternalID.graphql"
 import { AnalyticsSchema, useTracking } from "Artsy"
 import { RouterLink } from "Artsy/Router/RouterLink"
+import styled from "styled-components"
+import { Media } from "Utils/Responsive"
+import { ArtistConsignment } from "../Utils/getConsignmentData"
 import { LightPurpleColor, SectionContainer } from "./SectionContainer"
 
 interface ArtistConsignHeaderProps {
+  artistConsignment: ArtistConsignment
   artistName: string
+  artworksByInternalID: Consign_artworksByInternalID
 }
 
 export const ArtistConsignHeader: React.FC<ArtistConsignHeaderProps> = props => {
   const tracking = useTracking()
 
   return (
     <SectionContainer background={LightPurpleColor}>
+      <Media greaterThan="xs">
+        {classNames => {
```

Comment:
Media component creates its own wrapping component. Since we're doing some custom layout stuff, one can use the renderProps pattern to apply classes to your own component.

Reply:
I see what's happening here, that class names are being passed from the generated wrapper down into the box below, but I can't quite figure out why? Is it because you want some styling on both containers? 🤔

Reply:
the `<Media>` container doesn't accept a styles or attributes, but has certain layout properties. I need a subset of those layout properties for the media query aspect, but need to change others because we're doing some position absolute magic. So I grab the classNames, apply them to my own container, then augment my container with desired props.